test/fuzz/fuzz-unit-file/directives-all.service

```diff
@@ -408,6 +408,7 @@ Bridge=
 Broadcast=
 Cache=
 CacheFromLocalhost=
+CacheSize=
```

Comment:
Not a big matter, but this is not necessary. This is for unit files for service manager, e.g. .service, .socket, and so on. Please drop.